core/src/main/scala/org/apache/spark/deploy/security/HBaseDelegationTokenProvider.scala

```diff
@@ -50,12 +52,59 @@ private[security] class HBaseDelegationTokenProvider
       creds.addToken(token.getService, token)
     } catch {
       case NonFatal(e) =>
-        logWarning(s"Failed to get token from service $serviceName", e)
```

Comment:
Why dropped exception printing? It could still hold useful information if 2.x also failed.
Additionally wondering whether it's really a warning?

Reply:
yes better to convey the error to  user, i modified the message . thnks